web-stories.php

```diff
@@ -47,7 +47,7 @@
 define( 'WEBSTORIES_PLUGIN_DIR_PATH', plugin_dir_path( WEBSTORIES_PLUGIN_FILE ) );
 define( 'WEBSTORIES_PLUGIN_DIR_URL', plugin_dir_url( WEBSTORIES_PLUGIN_FILE ) );
 define( 'WEBSTORIES_ASSETS_URL', WEBSTORIES_PLUGIN_DIR_URL . 'assets' );
-define( 'WEBSTORIES_MINIMUM_PHP_VERSION', '5.6' );
+define( 'WEBSTORIES_MINIMUM_PHP_VERSION', '7.0' );
```

Comment:
There is a comment below saying 
> // Main plugin initialization happens there so that this file is still parsable in PHP < 5.6.

Should that be changed to `...parsable in PHP < 7.0` now?

Also, should the PHP 7.0 usage information be in the docs somewhere, too?

Reply:
Changing the header here will make the min requirement show on the plugin page. 

<img width="356" alt="Screenshot 2021-06-21 at 12 50 23" src="https://user-images.githubusercontent.com/237508/122757239-4e417100-d28f-11eb-97da-31fbe6c7bd6c.png">

It will also stop users from install the plugin, as it doesnt meet the min requirements.